client/extensions/zoninator/components/settings/zones-dashboard/zone-item.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
```

Comment:
Oh, that reminds me -- we should [start to](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) `import { PropTypes } from 'prop-types';`.